frontend/src/Pages/components/AdminCategories/EditCategoryWords.tsx

```diff
@@ -0,0 +1,168 @@
+import { useEffect } from "react";
+import { useForm } from "react-hook-form";
+import { useHistory, useParams } from "react-router-dom";
+import { fetchAdminQuizQuestion } from "../../../features/AdminCategories/adminCategoriesWordsAPI";
+
+const EditCategoryWords = () => {
+  const { quizId, questionId } =
+    useParams<{ quizId: string; questionId: string }>();
+  const history = useHistory();
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+    reset,
+  } = useForm();
+
+  useEffect(() => {
+    (async () => {
+      const question = await fetchAdminQuizQuestion(Number(questionId));
+      reset({
+        word: question.data.word,
+        choices: [
+          {
+            value: question.data.choices[0].value,
+            is_correct: question.data.choices[0].is_correct,
+          },
+          {
+            value: question.data.choices[1].value,
+            is_correct: question.data.choices[1].is_correct,
+          },
+          {
+            value: question.data.choices[2].value,
+            is_correct: question.data.choices[2].is_correct,
+          },
+          {
+            value: question.data.choices[3].value,
+            is_correct: question.data.choices[3].is_correct,
+          },
+        ],
+      });
+    })();
+  }, [questionId, reset]);
+
+  const onSubmit = async (data: {
+    word: string;
+    choices: { value: string; is_correct: boolean }[];
+  }) => {
+    // await saveAdminQuizQuestions(data, Number(quizId));
```

Comment:
Is this intentionally commented?

Reply:
Yes sir, I implemented that on the next pull request.